protocols/1181-laboratory-specialists-inc/mic_dilution_prep.ot2.py

```diff
@@ -0,0 +1,114 @@
+from opentrons import labware, instruments
+
+plate = labware.load('96-flat', '1')
+trough = labware.load('trough-12row', '5')
+dilution_plate = labware.load('96-deep-well', '4')
+
+# reagent setup
+buffer = trough.wells('A1')
+stock = trough.wells('A2')
+
+tiprack10 = labware.load('tiprack-10ul', '6')
+tiprack300 = labware.load('tiprack-200ul', '7')
+
+m10 = instruments.P10_Multi(
+    mount='right',
+    tip_racks=[tiprack10])
+
+m300 = instruments.P300_Multi(
+    mount='left',
+    tip_racks=[tiprack300])
+
+
+def serial_dilution_calculations(
+        stock_conc,
+        first_conc,
+        dilution_factor,
+        dilution_num,
+        dilution_loc):
+
+    """
+    Return the following lists:
+    source_wells - locations of source well
+    dil_wells - locations of dilution well
+    source_vol - volumes of dilution source for each well
+    buffer_vol - volumes of buffer for each well
+    """
+    prep_vol = 50 * 6 + 100  # Prepare more than enough volume for 6 replicates
+    final_concs = [0] + [first_conc/(2**repeat)
```

Comment:
hmmm you might need to explain this to me in person. A bit confused on a few different things